pages/profile/index.tsx

```diff
@@ -0,0 +1,65 @@
+import BottomNav from '../../components/BottomNav'
+import styles from '../../styles/Page.module.css'
+import profileStyles from '../../styles/Profile.module.css'
+import TopNav from '../../components/TopNav'
+import MenuButton from '../../components/profile/MenuButton'
+import star from '../../public/assets/icon/star-outlined.svg'
+import share from '../../public/assets/icon/share.svg'
+import edit from '../../public/assets/icon/edit.svg'
+import settings from '../../public/assets/icon/settings.svg'
+import file from '../../public/assets/icon/file.svg'
+import shield from '../../public/assets/icon/shield.svg'
+import logout from '../../public/assets/icon/logout.svg'
+import ProfilePicture from '../../components/profile/ProfilePicture'
+import React, { useState } from 'react'
+
+// TO-DO: logout and share app needs to be handled differently from the other buttons
+const menuButtonsDict = [
+  [star, 'Saved Support', 'saved-support', 'block-button-container'],
+  [share, 'Share this App', 'share-this-app', 'block-button-container'],
+  [edit, 'Edit Account Profile', 'edit-profile', 'slim-button-container'],
+  [settings, 'Settings', 'settings', 'slim-button-container'],
+  [
+    file,
+    'Terms and Conditions',
+    'terms-and-conditions',
+    'slim-button-container'
+  ],
+  [shield, 'Privacy Policy', 'privacy-policy', 'slim-button-container'],
+  [logout, 'Logout', 'logout', 'slim-button-container']
+]
+
+export default function Profile() {
+  // dummy data
+  const [name, setName] = useState('Rachel Tan')
+  const [phoneNumber, setPhoneNumber] = useState('9128 3271')
```

Comment:
I'm confused. I'm trying to demo this on Gitpod and I don't see where `setPhoneNumber` is called. I can't seem to find edit profile page. I feel like I'm missing somethign.